WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderTracker.kt

```diff
@@ -0,0 +1,35 @@
+package org.wordpress.android.ui.reader
+
+import org.wordpress.android.ui.reader.utils.ReaderTrackersProvider
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class ReaderTracker @Inject constructor(trackersProvider: ReaderTrackersProvider) {
+    // we did not protect the concurrent access to the trackers
+    // intended usage is to call the ReaderTracker fun from the UI thread
```

Comment:
Yeah...the @MainThread annotation sounds good! 😎 I actually partially kept the comment since what I had in mind was to make the fact that we sync access toward UI thread transparent to caller code. Probably we can follow up on this when extending with the IA related tracking. Wdyt?

Reply:
Ohh, I see. 👍 Sounds good!